### PR TITLE
add: sdk branch

### PR DESCRIPTION
**Pull request by Ankit098**

### CODE REVIEW

File: src/test/java/runners/BstackRunner.java [hunk 1]
```diff
@@ -65,21 +30,14 @@ public Stream<TestTemplateInvocationContext> provideTestTemplateInvocationContex
...
+            mainConfig = (JSONObject) parser
+                        .parse(new FileReader("src/test/resources/conf/" + System.getProperty("config")));
             platformConfig = (JSONArray) mainConfig.get("environments");
-            commonCapsConfig = (Map<String, Object>) mainConfig.get("capabilities");
+            wss = (String) mainConfig.get("wss");
 
             for (int i = 0; i < platformConfig.size(); i++) {
                 JSONObject platform = (JSONObject) platformConfig.get(i);
-                platform.putAll(commonCapsConfig);
-                platform.put("browserstack.username", username);
```
Comment (karanshah-browserstack) [inline src/test/java/runners/BstackRunner.java:74]: Fetch username & accesskey from env var or yml file so if user doesn't have IDE plugin install and runs via IDE it at least trigger a runs on BS.

File: src/test/resources/conf/browserstack.conf.json [hunk 2]
```diff
@@ -1,14 +1,5 @@
...
-    "project": "BStack Project",
-    "build": "browserstack-build-1",
-    "buildTag": "Regression",
-    "browserstack.debug": "true",
-    "browserstack.console": "info"
-  },
+  "wss": "ws://127.0.0.1:64225/ec873d261d8cad986db668660c8ab412",
   "environments": [
     {
       "os": "Windows",
```
Comment (karanshah-browserstack) [inline src/test/resources/conf/browserstack.conf.json]: Get Rid of this file as all caps will be governed through yml file

**Review by karanshah-browserstack — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 3]
```diff
@@ -63,10 +64,51 @@
...
+                    </argLine>
+                    <testFailureIgnore>false</testFailureIgnore>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
     <profiles>
         <profile>
-            <id>sample-local-test</id>
+            <id>local</id>
```
Comment (karanshah-browserstack) [inline pom.xml]: sample-local-test

File: pom.xml [hunk 4]
```diff
@@ -75,28 +117,19 @@
...
+                                -javaagent:${com.browserstack:browserstack-java-sdk:jar}
+                            </argLine>
                         </configuration>
                     </plugin>
                 </plugins>
             </build>
         </profile>
         <profile>
-            <id>sample-test</id>
+            <id>sample</id>
```
Comment (karanshah-browserstack) [inline pom.xml]: sample-test

**Review by karanshah-browserstack — APPROVED** [review]

File: pom.xml [hunk 5]
```diff
@@ -17,6 +17,7 @@
         <httpclient>4.5.13</httpclient>
         <parallel.count>3</parallel.count>
         <tests.single>**/tests.SampleTest.java</tests.single>
```
Comment (karanshah-browserstack) [inline pom.xml]: Rename to tests.sample

**Review by karanshah-browserstack — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 5 above)
Comment (Ankit098) [reply pom.xml]: done

File: pom.xml [hunk 6]
```diff
@@ -63,7 +64,48 @@
...
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${maven-surefire-plugin}</version>
+                <configuration>
+                    <includes>
+                        <include>${tests.single}</include>
```
Comment (karanshah-browserstack) [inline pom.xml]:
```suggestion
                        <include>${tests.sample}</include>
```

**Review by karanshah-browserstack — CHANGES_REQUESTED** [review]

**Review by karanshah-browserstack — APPROVED** [review]

**Pull request merged**